File: atcoder/ABC180-189/ABC180_2020-10-17/B.rb
n = gets
x = gets.split(' ')

y = 0
m = 0
c = 0

while n > 0
	m += abs(x[n])
	y += x[n]*x[n]
	if c < x[n]
		x = x[n]
	n -= 1;
end

y = sqrt(y)

print(m)
print("%15.f",n)
print(c)

File: atcoder/ABC180-189/ABC180_2020-10-17/B.py
import math

n = int(input())
# x = []
# for i in range(n):
# 	x[i] = int(input())
x = list(map(int, input().split(" ")))

m = 0
c = 0
y = 0.0
for j in range(0, len(x) - 1):
	if m >= 0:
		m += x[j]
	else:
		m -= x[j]
	y += x[j] * x[j]
	if c < x[j]:
		x = x[j]

y = math.sqrt(y)

print(m)
print(y)
print(c)
